src/main/js/dynamic/loaded/certificates/CertificatesPage.js

```diff
@@ -91,6 +91,14 @@ class CertificatesPage extends Component {
     createModalFields() {
         const { certificate } = this.state;
         const { fieldErrors } = this.props;
+
+        const fieldStatuses = fieldErrors.fieldErrors;
```

Comment:
This code could most likely be moved into the httpErrorUtilities.js file to be used elsewhere until we convert to using the validation rest endpoint with a better object.

Reply:
Yeah, this was actually the line I was concerned about. Not sure how it was working before, but I think the best solution might be to rename the prop for now so we don't have the weird nesting.